data/migrations/20201013113404_groomers_customers_services_pets.js

```diff
@@ -0,0 +1,20 @@
+exports.up = (knex) => {
+  return (
+    knex.schema
+      // .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
```

Comment:
Maybe remove comment for file cleanliness

Reply:
You are right. Done :)